package.json

```diff
@@ -31,6 +36,7 @@
     "express": "^4.17.1",
     "morgan": "^1.10.0",
     "multer": "^1.4.3",
+    "multer-s3": "^2.10.0",
```

Comment:
Didn't end up using multer-s3, but plan on using if I have time to refactor and make my files less ugly.